backend/tests/unit/core/schema/test_attribute_parameters.py

```diff
@@ -260,3 +260,48 @@ def test_validate_min_max_text_attribute() -> None:
         TextAttributeParameters(min_length=10, max_length=5)
 
     assert config.SETTINGS.main.schema_strict_mode
+
+
+def test_list_attribute_with_regex_parameter() -> None:
```

Comment:
I would be good to add a test to ensure that the regex is properly applied to the values in the list